clang/Util.d

```diff
@@ -112,7 +122,35 @@ File namedTempFile(string prefix, string suffix)
     scope (exit) close(fd);
 
     if (fd == -1)
-        throw new NamedTempFileException("Cannot create \"%s\" temporary file.".format(path));
+    {
+        auto message = format("Cannot create temporary file \"%s\".", path);
```

Comment:
No, I mean to store the path as an instance variable in `NamedTempDirException`. It should be possible to retrieve the path without having to parse the message.